### PR TITLE
[READY] - nix-shell and kermit configs

### PR DESCRIPTION
**Pull request by sarcasticadmin**
## Description of PR
For a discussion and intro of nixpkgs into scale-network

## Previous Behavior
- No kermit config
- No shell.nix

## New Behavior
- Kermit config for junipers
- shell.nix with basic tools

## Tests
How was this PR tested?

### CODE REVIEW

File: shell.nix [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# TODO: Pin this to a specific version of nixpkgs
```
Comment (sarcasticadmin) [inline shell.nix:1]: Needs a lot of work but this is a decent start :)

Comment (kylerisse) [reply shell.nix:1]: One small step for some. One giant leap for us.

**Review by kylerisse — APPROVED** [review]
Thanks @sarcasticadmin

File: shell.nix (hunk 1 above)
Comment (sarcasticadmin) [reply shell.nix:1]: hahaha so true :laughing:

**Pull request merged**